packages/libs/error-stack/README.md

```diff
@@ -111,3 +111,13 @@ Caused by:
 ```
 
 Please see the [documentation] for a full description.
+
+## Emacs [rust-mode](https://github.com/rust-lang/rust-mode) workaround.
```

Comment:
Can we instead add a *Troubleshooting* sub-section and add this as a sub-sub-section below that?

Reply:
+1 for this. Happy then to approve. Thanks!

Reply:
```suggestion
## Troubleshooting

### Emacs [rust-mode](https://github.com/rust-lang/rust-mode) workaround.

```